hub.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gradle:jdk8 as builder
+
+COPY . /home/gradle/src
+USER root
```

Comment:
is it necessary to use root user?

Reply:
Yes it is, 
Can't quite remember the issue but something to do with `COPY` uses the root user and gradle refuses to run if its owned by ROOT, there's a COPY --user flag but the docker hub builders don't support the user flag so you have to do this hack instead